Makefile

```diff
@@ -1156,7 +1156,7 @@ lint: lint-api lint-go lint-kube-agent-updater lint-tools lint-protos lint-no-ac
 # Similar to lint.
```

Comment:
Can you update this comment block please? It looks like it's grossly outdated.

I think it should say something in the lines of "Runs linters without dedicated GitHub Actions."

Reply:
done